src/Epic/index.tsx

```diff
@@ -151,20 +161,24 @@ export const Epic: React.FC<EpicProps> = (props: EpicProps) => {
                         </p>
                     ))}
                     {/* buttonText and buttonUrl will have been checked for not undefined in canRenderEpic */}
-                    <ContributionButtonWithReminder
-                        buttonText={buttonText as string}
-                        buttonUrl={buttonUrl as string}
-                        hidePaymentIcons={hidePaymentIcons}
-                        remindMeButtonText={remindMeButtonText}
-                        remindMeConfirmationText={remindMeConfirmationText}
-                        remindMeConfirmationHeaderText={remindMeConfirmationHeaderText}
-                        trackClick={(buttonId: number) =>
-                            trackClick({
-                                internalButtonId: buttonId,
-                                ophanComponentId: ophanComponentId as string,
-                            })
-                        }
-                    ></ContributionButtonWithReminder>
+                    <div css={styles.buttonWrapperStyles}>
+                        <ContributionCtaButton
+                            buttonText={buttonText as string}
+                            buttonUrl={buttonUrl as string}
+                            hidePaymentIcons={hidePaymentIcons as string}
+                            ophanComponentId={ophanComponentId as string}
+                            trackClick={trackClick}
+                        />
+                        {reminderStage && reminderOption && (
```

Comment:
Sorry I didn't notice this before - is it right that a `reminderOption` should always be specified when scheduling a reminder? I think in SDC, `reminderOption` isn't usually specified.

Reply:
Good spot. Removed!